src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -197,18 +198,17 @@ describe('Exam assessment', () => {
                 cy.wait(examEnd.diff(dayjs(), 'ms'));
             }
             cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
-            const score = '5 of 10 points';
             // Sometimes the feedback fails to load properly on the first load...
-            cy.reloadUntilFound(`jhi-result:contains(${score})`);
-            cy.get('jhi-result').contains(score).should('be.visible');
+            cy.reloadUntilFound(`#result-score`);
+            cy.get('#result-score').should('contain.text', '5 of 10 points');
```

Comment:
Only declare the selector once and assert the visibility
```suggestion
            const resultSelector = '#result-score';
            cy.reloadUntilFound(resultSelector);
            cy.get(resultSelector).should('contain.text', '5 of 10 points').and('be.visible');
```